scala/tempo/src/test/scala/com/databrickslabs/tempo/EMATests.scala

```diff
@@ -34,12 +35,21 @@ class EMATests
 		                                    Row("S2", "2020-09-01 00:19:12", 32.0, 21.0)),
 	                                  "event_ts" )
 
-	it("EMA test") {
+	it("EMA method") {
 
 		val tsdf = TSDF(df, tsColumnName = "event_ts", partitionColumnNames = Seq("symbol"))
 		val emaDf = tsdf.EMA("trade_pr", window = 2, exp_factor = 0.5).df
 
 		assert(emaDf.collect().sameElements(dfExpected.collect()))
 	}
 
+	it("EMA Transformer"){
+		val tsdf = TSDF(df, tsColumnName = "event_ts", partitionColumnNames = Seq("symbol"))
+
+		val ema_tx = EMATX("trade_pr","ema_trade_pr",2,0.5)
```

Comment:
Just for clarity, we have inputCols, outputCols, windowLength and alpha - can we name these in the test?

Reply:
Do you mean: 

- use named parameters in the constructor call? Or
- have multiple test cases for each parameter?

Reply:
I've added in the named parameters in the test code